src/OrdinaryDiffEq.jl

```diff
@@ -177,7 +177,7 @@ module OrdinaryDiffEq
 
   export ImplicitEuler, ImplicitMidpoint, Trapezoid, TRBDF2, SDIRK2, Kvaerno3,
          KenCarp3, Cash4, Hairer4, Hairer42, SSPSDIRK2, Kvaerno4, Kvaerno5,
-         KenCarp4, KenCarp5
+         KenCarp4, KenCarp5, RK4Threaded
```

Comment:
this is a bad name since it's not RK4 threaded...

Reply:
Will `DIRK44_2`(Diagonal Implicit RK 4th order, 4 stage, 2 parallel) be okay?

Reply:
what's the name of the author the tableau comes from?

Reply:
Iserles and Nørsett, 1990

Reply:
Yeah, so I'd call it like IserNor4